chainerx_build_helper.py

```diff
@@ -102,10 +104,17 @@ def config_setup_kwargs(setup_kwargs, build_chainerx):
     setup_kwargs['package_data'] = {
         'chainerx': ['py.typed', "*.pyi"],
     }
+
+    # TODO(durswd): Remove build_targets. Because build_targets is used without Windows
+    if platform.system() == "Windows":
+        build_targets = ['_core.pyd']
+    else:
+        build_targets = ['_core.so']
+
     setup_kwargs.update(dict(
         cmdclass={'build_ext': CMakeBuild},
         ext_modules=[CMakeExtension(
             name='chainerx._core',
-            build_targets=['_core.so'],
+            build_targets=build_targets,
```

Comment:
I think it's good to leave the comments above but actually, we don't need this change since the property is ignored anyway for Windows.